packages/expo-splash-screen/README.md

```diff
@@ -297,9 +331,110 @@ The newly created `SplashScreen.storyboard` needs to be marked as the `Launch Sc
+
+You can use this online online generator to obtain 1x1px `.png` images with desired colors: http://www.1x1px.me.
+
+1. Create `SplashScreenBackground` `Image Set` with desired background colors for each mode in your `Images.xcassets` directory:
+    - open your `.xcassets` directory with images,
+    - in the content panel add `New image set` and name it `SplashScreenBackground`,
+    - convert this `Image set` to support `Dark Appearance` by navigating to `Attributes Inspector` in the right panel and changing `Appearance` to `Any, Dark`,
+    - provide images with colors for every mode (you can generate color 1x1px images using http://www.1x1px.me).
+
+<details>
+ <summary>Show image with details</summary>
+<img src="./assets/configuration-ios-addSplashScreenBackgroundImages.png" height="250" />
+</details>
+
+2. Update `SplashScreen.storyboard` to consist of single top-level `View` with to `Image View` subviews (solid-colored image in the background and actual splash screen image in the foreground):
+    - open `SplashScreen.storyboard` and replace `Image View` with a plain `View` (search `Library` for it and drag&drop it in place of current `Image View`),
+    - add two 
+
+3. Configure first `Image View` (background color):
+    - configure attributes in `Attributes Inspector`:
+        - set `Image` to `SplashScreenBackground` (prepared in previous step),
+        - set `Content Mode` to `Scale To Fill` (color needs to take all available space),
+    - make this subview take all available space in parent view:
+        - open `Add new constraints` bottom menu,
+        - make sure `Constrain to margin` is not checked,
+        - for every input open dropdown and select `View` (parent view reference) and set `0` as the value,
+        - once every side is covered (`0` value and parent view reference selected in dropdown) hit `Add 4 Constraints`,
+        - observe that in `View Hierarchy Inspector` constraints are added and `Image View` resized to take whole place of parent view.
+
+<details>
+ <summary>Show image with details</summary>
+<img src="./assets/configuration-ios-createBackgroundImageView.png" height="350" />
+</details>
+
+4. Configure second `Image View` (actual splash screen image):
+    - select second `Image View` and select correct `Image` in `Attributes Inspector` alongside with desired `Content Mode`,
+    - make this subview take all available space in parent view (see previous step).
+     
+<details>
+ <summary>Show image with details</summary>
+<img src="./assets/configuration-ios-addConstraintsToImageViews.png" height="350" />
+</details>
+
+##### Provide different splash screen image
 
-### 🤖 Configure Android
+You might want to add separate image for `dark` mode. If the system is switched to `dark` mode it would pick this different image instead of normal one and present it in splash screen view.
```

Comment:
```suggestion
You might want to add a separate image for `dark` mode. If the system is switched to `dark` mode, it would pick this different image instead of the normal one and present it in the splash screen view.
```